config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigquerydatatransferconfigs.bigquerydatatransfer.cnrm.cloud.google.com.yaml

```diff
@@ -6,20 +6,16 @@ metadata:
   creationTimestamp: null
   labels:
     cnrm.cloud.google.com/managed-by-kcc: "true"
-    cnrm.cloud.google.com/stability-level: alpha
     cnrm.cloud.google.com/system: "true"
-    cnrm.cloud.google.com/tf2crd: "true"
```

Comment:
This means we need to provide the direct controller, but I don't see it in this PR. Since this is a alpha resource, could you add 
 `labels="cnrm.cloud.google.com/tf2crd=true"` in the types.go? Example  
https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/660ef0bdad70d384e0c742a4222d97743c582aaf/apis/kms/v1beta1/kmskeyring.go#L73

I'll update the guide to capture this.

Reply:
Actually, you can add this resource to the `excluded_resources` to not override the CRD until the controller is ready (especially since this resource has API changes). https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/dev/tasks/generate-crds#L58